source/ways-of-working/asking-for-help/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Asking for help
+last_reviewed_on: 2020-05-04
+review_in: 6 months
+weight: 3
```

Comment:
Should we change this so that it goes at end of documentation ? not sure where it is shown currently.

Reply:
yep, will increase weight to put it at the bottom